code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -110,7 +112,7 @@
 /datum/gas_reaction/miaster/init_reqs()
 	requirements = list(
 		/datum/gas/miasma = MINIMUM_MOLE_COUNT,
-		"MIN_TEMP" = MIASTER_STERILIZATION_TEMP
+		"MIN_TEMP" = MIASTER_STERILIZATION_TEMP,
```

Comment:
Why add a `,` in the end of every list?

Reply:
It's a code style thing that some maints insist on for lists defined over multiple lines.

Reply:
Well is not enforced, but whatever